FHEM/90_SIGNALduino_un.pm

```diff
@@ -16,296 +16,267 @@ use POSIX;
 use List::Util qw(any);         # for any function
 use lib::SD_Protocols;          # for any function
 
-my @bitcountlength = (0,0,0);		# array min|default|max
+my @bitcountlength = (0,0,0);   # array min|default|max
 
 #####################################
-sub
-SIGNALduino_un_Initialize($)
-{
+sub SIGNALduino_un_Initialize {
   my ($hash) = @_;
 
-
   $hash->{Match}     = '^[u]\d+(?:.\d)?#.*';
-  $hash->{DefFn}     = "SIGNALduino_un_Define";
-  $hash->{UndefFn}   = "SIGNALduino_un_Undef";
-  $hash->{AttrFn}    = "SIGNALduino_un_Attr";
-  $hash->{SetFn}     = "SIGNALduino_un_Set";
-  $hash->{ParseFn}   = "SIGNALduino_un_Parse";
-  $hash->{AttrList}  = "IODev do_not_notify:0,1 stateFormat showtime:0,1 ignore:0,1 ".$readingFnAttributes;
+  $hash->{DefFn}     = 'SIGNALduino_un_Define';
```

Comment:
Lass uns hier doch gleich coderefs hinterlegen. Ist die sauberste Methode

Reply:
Ich stehe hier gerade etwas auf dem Schlauch. *sorry*
Was genau meinst du?

Reply:
Gleiches vorgehen wie hier und ff:

https://github.com/RFD-FHEM/RFFHEM/blob/f7ca7bf284cb6bca22ff81f4d9d1b573bdf2b592/FHEM/00_SIGNALduino.pm#L286

Reply:
umgesetzt in https://github.com/RFD-FHEM/RFFHEM/pull/914/commits/0aae01a80a4c347f20e2215d49b7b454a399ac8b